react/features/base/react/components/NavigateSectionList.js

```diff
@@ -334,11 +326,11 @@ class NavigateSectionList extends Component<Props> {
      */
     _renderSection(section) {
         return (
-            <View style = { styles.listSection }>
+            <Container style = { styles.listSection }>
                 <Text style = { styles.listSectionText }>
-                    { section.section.title }
+                    {section.section.title}
```

Comment:
Hmmm... I was thinking eslint requires the spaces in this use cases, isn't it?

Reply:
Not sure why it didnt catch that one. I changed back.